### PR TITLE
Feature/users list

### PR DESCRIPTION
**Pull request by andreswolox**
## Summary  [Card](https://trello.com/c/nzafbFcv)

- **IMPLEMENTATIONS**
 Created endpoint `/users` GET.
 Created `getAll` Controller function.
 Created `getAll` Service function.
 Created `session` middleware.
 Created `getAll` Repository function.
 Created `tokenSchema` Schema.
 Moved`Wolox domains` to `envinronments`.
 Created `Tests` for the endpoint and service.

### CODE REVIEW

File: app/utils/constants.js [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-exports.DOMAIN_WOLOX_CO = `wolox.${exports.DOMAIN_COLOMBIA}`;
-exports.DOMAIN_WOLOX_CL = `wolox.${exports.DOMAIN_CHILE}`;
-exports.DOMAIN_WOLOX_MX = `wolox.${exports.DOMAIN_MEXICO}`;
+exports.DOMAIN_WOLOX_AR = 'wolox.com.ar';
+exports.DOMAIN_WOLOX_CO = 'wolox.co';
+exports.DOMAIN_WOLOX_CL = 'wolox.cl';
+exports.DOMAIN_WOLOX_MX = 'wolox.com.mx';
 
 // EMAILS
 exports.EMAIL_WOLOX_AR = `@${exports.DOMAIN_WOLOX_AR}`;
```
Comment (jersonlopez) [inline app/utils/constants.js:10]: What is the use for these constants?

File: config/development.js [hunk 2]
```diff
@@ -10,7 +10,8 @@ exports.config = {
     },
     jsonPlaceHolderApi: {
       baseUrl: process.env.JSON_PLACE_HOLDER_API_BASE_URL
-    }
+    },
+    availableDomains: process.env.AVAILABLE_DOMAINS
```
Comment (jersonlopez) [inline config/development.js:14]: don't forget to update the `env.example` file, please

File: app/utils/constants.js (hunk 1 above)
Comment (andreswolox) [reply app/utils/constants.js:10]: These constants are currently used in email's schema validations.

File: config/development.js (hunk 2 above)
Comment (andreswolox) [reply config/development.js:14]: Updated it! :)

**Review by jersonlopez — APPROVED** [review]
🎉 🎊 🎉

**Pull request merged**